### PR TITLE
Adjustments to dev tools.

### PR DESCRIPTION
**Pull request by LegacyValken**

### CODE REVIEW

File: lua/swamp/dev/cl_devtools.lua [hunk 1]
```diff
@@ -17,16 +17,18 @@ local allowed = {
     ["STEAM_0:1:33536503"] = true, -- medroit
     ["STEAM_0:0:16678862"] = true, -- legacy
     ["STEAM_0:0:183303619"] = true -- pura
-    
 }
 
+local player = FindMetaTable("Player")
+function player:IsLocalDev()
```
Comment (swampservers) [inline lua/swamp/dev/cl_devtools.lua]: the global "Player" refers to this metatable already

File: lua/swamp/dev/cl_devtools.lua [hunk 2]
```diff
@@ -183,13 +184,35 @@ concommand.Add("dev_refresh", function(_, _, args)
...
 concommand.Add("trace", function(ply)
+    if not Me:IsLocalDev() then
+        print( "Not allowed!" )
+        return
+    end
     print(Me:GetEyeTrace().HitPos)
 end)
 
 concommand.Add("origin", function(ply)
+    if not Me:IsLocalDev() then
```
Comment (swampservers) [inline lua/swamp/dev/cl_devtools.lua]: there is really no point to blocking this stuff

Comment (swampservers) [reply lua/swamp/dev/cl_devtools.lua]: all it does is potentially make it harder for someone to contribute

Comment (LegacyValken) [reply lua/swamp/dev/cl_devtools.lua]: I didn't consider this, thanks for that.

**Comment by LegacyValken** [comment]
Forgot to ask for another review.

**Pull request merged**